src/util/loanSearch/queryParamUtils.js

```diff
@@ -97,7 +127,10 @@ export async function applyQueryParams(apollo, query, allFacets, queryType, page
 		countryIsoCode: getCountryIsoCodesFromQueryParam(query.country || query.countries, allFacets),
 		sectorId: getIdsFromQueryParam(query.sector, allFacets.sectorNames, allFacets.sectorFacets),
 		sortBy: queryType === FLSS_QUERY_TYPE ? lendToFlssSort.get(query.sortBy) : query.sortBy,
-		themeId: getIdsFromQueryParam(query.attribute || query.attributes, allFacets.themeNames, allFacets.themeFacets),
+		themeId: getIdsFromQueryParam(
+			query.attribute || query.attributes || query.theme,
```

Comment:
Check `query.theme` on incoming query params. This will be converted to attribute ids!